### DISCUSSION TITLE
About web-vnc?

### DISCUSSION THREAD

**Discussion by mremptypants** [Ylianst/MeshCentral#5946] [General]
Do I need to install a vnvserver one the client?  My guess it should jsut work out the box right?

**Comment by si458** [comment]
yes to use the Web-VNC, the remote device must have a VNC server of some form setup/installed

**Reply by mremptypants** [reply]
thought so just was confused. One computer worked out the box and the other one does not.  Sorry about the wrong topic too..was working to get tigervnc to work just could not get it to connect, thanks again.

**Reply by si458** [reply]
no worries! just check tigervnc is using password auth and running on port 5900, but should work out the box

**Comment by mremptypants** [comment]
For people that see this, it looks like something was running on vnc:0,I also had to pass x11 in sshd file for my case, I did turn off localhost in the config file.
